app/controllers/models_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ModelsController < ApplicationController
   include Foreman::Controller::AutoCompleteSearch
   before_filter :find_resource, :only => [:edit, :update, :destroy]
+  before_filter :find_hosts_count, :only => :index
```

Comment:
maybe create a mixin for all controllers that need a count? or consider changing it to all belongs_to attribute a host has?